### DISCUSSION TITLE
Enable symbols in usernames

### DISCUSSION THREAD

**Discussion by Mehdical-Man** [stoatchat/discussions#589] [Feature Suggestions]
If I wanted to create a username like "Not So Cliché" (without the quotes), I can only use an underscore as a space, resulting in "Not_So_Cliché", which can be quite limiting. Discord had a feature where you could spaces in your username.

**Comment by Rexogamer** [comment]
We intentionally don't allow spaces and other characters in usernames to make them easier to type/handle - however, there are no restrictions on what characters you can use in display names (assuming you want your name to show as `Not So Cliché` to other users).
